superset/models/reports.py

```diff
@@ -74,6 +74,12 @@ class ReportDataFormat(str, enum.Enum):
     DATA = "CSV"
 
 
+class ReportCreationMethodType(str, enum.Enum):
+    CHARTS = "charts"
+    DASHBOARD = "dashboard"
```

Comment:
nit, but this is the only singular value

Reply:
would you prefer that they all be plural or singular?

Reply:
changed all to plural.